.github/workflows/ci_ubuntu.yml

```diff
@@ -36,6 +36,13 @@ jobs:
               cxx: "g++",
               cflags: "-msse2"
             }
+          - {
+              name: "Ubuntu 22.04 ARM64 GCC",
+              os: ubuntu-22.04-arm,
+              cc: "gcc",
+              cxx: "g++",
+              cflags: ""
```

Comment:
i think you can omit the line if there's nothing to add

Reply:
Not sure. Let's try. :)